src/Symfony/Bridge/Doctrine/Tests/Security/RememberMe/DoctrineTokenProviderPostgresTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Symfony\Bridge\Doctrine\Tests\Security\RememberMe;
+
+use Doctrine\DBAL\Configuration;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Schema\DefaultSchemaManagerFactory;
+use Doctrine\ORM\ORMSetup;
+use Symfony\Bridge\Doctrine\Security\RememberMe\DoctrineTokenProvider;
+
+/**
+ * @requires extension pdo_pgsql
+ * @group integration
+ */
+class DoctrineTokenProviderPostgresTest extends DoctrineTokenProviderTest
+{
+    protected function bootstrapProvider()
+    {
+        if (!$host = getenv('POSTGRES_HOST')) {
+            self::markTestSkipped('Missing POSTGRES_HOST env variable');
+        }
```

Comment:
Should this be done in `setupBeforeClass()`?

Reply:
You're right, that sounds more efficient.